internal/server/handlers.go

```diff
@@ -0,0 +1,314 @@
+	if err != nil {
+		return "", http.StatusBadRequest, fmt.Errorf(incorrectIPErroString, err)
+	}
+	uid, err := strg.Registration(ctx, user.Login, user.Password, ua, ip)
+	if err != nil {
+		var pgErr *pgconn.PgError
+		status := http.StatusInternalServerError
+		err = fmt.Errorf(gormError, err)
+		if errors.As(err, &pgErr) && pgErr.Code == pgerrcode.UniqueViolation {
+			status = http.StatusConflict
+			err = fmt.Errorf("user registrating duplicate error: '%s'", user.Login)
+		}
+		return "", status, err
+	}
+	token, err := middlewares.CreateToken(key, tokenLiveTime, uid, ua, user.Login, ip)
+	if err != nil {
+		return "", http.StatusInternalServerError, fmt.Errorf(tokenGenerateError, err)
+	}
+	return token, http.StatusOK, nil
+}
+
+// LoginFunc ...
+// @Tags Авторизация
+// @Summary Авторизация пользователя в микросервисе
+// @Accept json
+// @Param params body LoginPassword true "Логи и пароль пользователя в формате json"
+// @Router /user/login [post]
+// @Success 200 "Успешная авторизация"
+// @Header 200 {string} Authorization "Токен авторизации"
+// @failure 400 "Ошибка в теле запроса. Тело запроса не соответствует json формату"
+// @failure 401 "Логин или пароль не найден"
+// @failure 500 "Внутренняя ошибка сервиса".
+func LoginFunc(ctx context.Context, body, key []byte, remoteAddr, ua string,
+	strg Storage, tokenLiveTime int) (string, int, error) {
+	user, err := isValidateLoginPassword(body)
+	if err != nil {
+		return "", http.StatusBadRequest, err
+	}
+	ip, _, err := net.SplitHostPort(remoteAddr)
+	if err != nil {
+		return "", http.StatusBadRequest, fmt.Errorf(incorrectIPErroString, err)
+	}
+	uid, err := strg.Login(ctx, user.Login, user.Password, ua, ip)
+	if err != nil {
+		if errors.Is(err, gorm.ErrRecordNotFound) {
+			return "", http.StatusUnauthorized, fmt.Errorf("user not found in system. Login: '%s'", user.Login)
+		} else {
+			return "", http.StatusInternalServerError, fmt.Errorf(gormError, err)
+		}
+	}
+	token, err := middlewares.CreateToken(key, tokenLiveTime, uid, ua, user.Login, ip)
+	if err != nil {
+		return "", http.StatusInternalServerError, fmt.Errorf(tokenGenerateError, err)
+	}
+	return token, http.StatusOK, nil
+}
+
+// OrdersAddFunc ...
+// @Tags Заказы
+// @Summary Добавление номера заказа пользователя
+// @Accept json
+// @Param order body string true "Номер заказа"
+// @Security ApiKeyAuth
+// @Param Authorization header string false "Токен авторизации"
+// @Router /user/orders [post]
+// @Success 200 "Заказ уже был добавлен пользователем ранее"
+// @Success 202 "Заказ успешно зарегистрирован за пользователем"
+// @failure 400 "Ошибка в теле запроса. Тело запроса пустое"
+// @failure 401 "Пользователь не авторизован"
+// @failure 409 "Заказ зарегистрирован за другим пользователем"
+// @failure 422 "Номер заказа не прошёл проверку подлинности"
+// @failure 500 "Внутренняя ошибка сервиса".
+func OrdersAddFunc(ctx context.Context, order string, strg Storage) (int, error) {
```

Comment:
```suggestion
func AddOrders(ctx context.Context, order string, strg Storage) (int, error) {
```

другие хендлеры тоже лучше переименовать по этой модели

Reply:
+